docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -28,21 +28,21 @@ Each log entry contains four fields:
 - labels
 - extracted fields
 
-Whenever any of these change, a log message will be printed:
+Enable the inspection output using the `--inspect` command-line option.<br>
+The `--inspect` option can be used in combination with `--stdin` and `--dry-run`.
 
 ```bash
 cat my.log | promtail --stdin --dry-run --inspect --client.url http://127.0.0.1:3100/loki/api/v1/push
 ```
 
 ![screenshot](../inspect.png)
 
-It can be used in combination with `--stdin` and `--dry-run`.
-
-Additions will be highlighted in green, modifications in yellow, and removals in red.
+The output uses color to highlight changes. Additions are in green, modifications in yellow, and removals in red.
 
 If no changes are applied during a stage, that is usually an indication of a misconfiguration or undesired behavior.
 
-**Note**: the `--inspect` flag should _not_ be used in production.
+The `--inspect` flag should not be used in production, as the calculation of changes between pipeline stages impacts
+Promtail's performance.
```

Comment:
You and I know that the impact is a negative thing. . . but we can make that obvious for the reader by changing the wording slightly.  I suggest one of 2 options:
1. The `--inspect` flag should not be used in production, as the calculation of changes between pipeline stages negatively impacts
Promtail's performance.
2. The `--inspect` flag should not be used in production, as the calculation of changes between pipeline stages reduces
Promtail's performance.